site/en/docs/extensions/mv3/declare_permissions/index.md

```diff
@@ -64,6 +67,23 @@ The following is an example of the permissions part of a [manifest][doc-manifest
 }
 ```
 
+## Host permissions {: #host-permissions }
+
+Host permissions allow extensions to interact with the URLs [matching patterns][doc-match]. Some [Chrome APIs][api-ref] require host permissions in addition to their own API permission. Host permissions allow extensions to:
```

Comment:
```suggestion
Host permissions allow extensions to interact with the URL's [matching patterns][doc-match]. Some [Chrome APIs][api-ref] require host permissions in addition to their own API permission. Host permissions allow extensions to:
```

Reply:
>Host permissions allow extensions to:

Can you future proof this a bit? This is the kind of thing we'll forget to update when new host permissions are added. Specifically, I think you should replace this with a statement that goes something like "Host permissions are documented with the APIs that require them. A few examples are described below."